treescript/requirements/local.py38.txt

```diff
@@ -7,4 +7,7 @@
 #
 -r test.py38.txt
 
-# The following packages are considered to be unsafe in a requirements file:
+# WARNING: The following packages were not pinned, but pip requires them to be
+# pinned when the requirements file includes hashes. Consider using the --allow-unsafe flag.
+# pip
+# setuptools
```

Comment:
@escapewindow is this a concern?

Reply:
Might be. If it's a problem, we may need to change pin.sh to --allow-unsafe; not sure if that'll be for all or just for local or what.